digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/RoadLinkService.scala

```diff
@@ -313,19 +315,20 @@ object RoadLinkService {
         val boundingBoxFilter2 = OracleDatabase.boundingBoxFilter(bounds2)
 
         sql"""
-        select dr1_id, mml_id, to_2d(shape)
+        select dr1_id, mml_id, to_2d(shape), linkkityyppi
```

Comment:
We can remove `linkkityyppi` from `select` - clause and add `where` filter to sql where we return only road links that are of car traffic - type. With this we can also remove filter at line 331.